dnf/util.py

```diff
@@ -643,30 +643,16 @@ def _is_file_pattern_present(specs):
     return False
 
 
-def is_container():
+def is_bootc_host():
```

Comment:
This patch breaks API. It removes is_container() function. If that function was never meant to be public, fine. But if the name is_bootc_host() is also not public, then rename it to _is_bootc_host().

Reply:
AFAIK container() was never meant to be public, @dcantrell can you confirm?

Reply:
Yes, I believe that was my fault with the original patch.  These were never meant to be public functions so is_container() should have been _is_container().  Likewise, is_bootc_host() should be _is_bootc_host()

Reply:
fixed.